source/manual/alerts/rotate-old-aws-iam-key.html.md

```diff
@@ -0,0 +1,19 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Rotate old AWS IAM Key
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-11-04
+review_in: 6 months
+---
+
+GOV.UK uses AWS [IAM](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_users.html)
```

Comment:
Do we have to have an alerts page, or can we just link from Icinga to the manual page for how to do it? This feels like unnecessary duplication.

Reply:
I think we can link directly to the manual, but as every other alert links to a check I was following the convention...

Reply:
It's possible to directly link to the docs https://github.com/alphagov/govuk-puppet/pull/9820